transfermethodrepository/build.gradle

```diff
@@ -0,0 +1,100 @@
+apply from: "$rootProject.projectDir/android-library.gradle"
+
+dependencies {
+    implementation project(":common")
+
+    androidTestImplementation "androidx.test:rules:$testRulesVersion"
+    testImplementation "org.robolectric:robolectric:$robolectricVersion"
+    testImplementation "com.squareup.okhttp3:mockwebserver:$mockWebServerVersion"
+}
+
+
+def aarFile = file("$buildDir/outputs/aar/userrepository-$version" + ".aar")
```

Comment:
I think it shouldn't be userrepository here. Maybe we can use the project name variable?
`def aarFile = file("$buildDir/outputs/aar/$project.name-$version" + ".aar")`